internal/hud/server/logs_reader.go

```diff
@@ -67,28 +82,36 @@ func (ls *LogStreamer) Handle(v proto_webview.View) error {
 	}
 
 	segments := v.LogList.Segments
-	if fromCheckpoint < ls.checkpoint {
+	if fromCheckpoint < ls.checkpointExternal {
 		// The server is re-sending some logs we already have, so slice them off.
-		deleteCount := ls.checkpoint - fromCheckpoint
+		deleteCount := ls.checkpointExternal - fromCheckpoint
 		segments = segments[deleteCount:]
 	}
 
-	// TODO(maia): filter for the resources that we care about (`tilt logs resourceA resourceC`)
-	//   --> and if there's only one resource, don't prefix logs with resource name?
+	// TODO(maia): if there's only one resource, don't prefix logs with resource name
```

Comment:
i wonder if it makes more sense to have an options object to ContinuingLines that takes a set of resources to filter on....we already do this internally with toLogLines logOptions (which filters on a set of spans)